docs/source/workflow_inputs.rst

```diff
@@ -650,11 +658,17 @@ The column names available in the schedule CSV files are:
   ``general_water_use``             frac     General water use internal gains.                                                  No
   ``heating_setpoint``              F        Thermostat heating setpoint schedule.                                              No
   ``cooling_setpoint``              F        Thermostat cooling setpoint schedule.                                              No
+  ``hvac_maximum_power_ratio``      frac     Variable speed system maximum power ratio schedule. [#]_                           No
   ``water_heater_setpoint``         F        Water heater setpoint schedule.                                                    No
   ``water_heater_operating_mode``   0/1      Heat pump water heater operating mode schedule. 0=hybrid/auto, 1=heat pump only.   No
   ``battery``                       -1 to 1  Battery schedule. Positive for charging, negative for discharging.                 No
   ================================  =======  =================================================================================  ===============================
 
+  .. [#] This feature is an advanced research capability. This schedule allows modeling shedding controls for variable speed HVAC systems (instead of setpoint changes) to limit the power of HVAC per `AHRI 1380 <https://www.ahrinet.org/search-standards/ahri-1380-i-p-demand-response-through-variable-capacity-hvac-systems-residential-and-small>`_. 
+         While any value can be entered, this is primarily intended to reflect the AHRI capabilities, which has two levels of load shed: "General Curtailment" and "Critical Curtailment". 
+         A "General Curtailment" signal prevents the equipment from running at more than 70% of max power and "Critical Curtailment" limits it to 40% of max power until comfort constraints are violated.
+         Values of 0.4 or below in the schedule are treated as "Critical Curtailment" events and values between 0.4-0.7 are treated as "General Curtailment" events.
```

Comment:
What happens when a value above 0.7 is provided? Also, should we say anything about the fact that any supplemental heating (integrated or separate) will continue to operate?

Reply:
> Also, should we say anything about the fact that any supplemental heating (integrated or separate) will continue to operate?

Note that this comment is specific to setpoint recovery. The question at hand is whether we should document how the backup heating should operate either during A) shedding events or B) setpoint recovery.

Reply:
From the current docs, the `AHRI 1380` seems to only have 0.4 (critical), 0.7 (general) and 1.0 (no shedding) these three scenarios being sent. But our workflow implementation is more general than that, which accepts any fraction value. My interpretation is that Jeff is recommending the standard values that this feature is mainly for, I can add some more explanation here for the actual implementation.